### ISSUE TITLE
(DESIGN-161) As an employee I want to have conversations with my supervisor once every four months, at minimum.

### ISSUE THREAD

**Issue by phoebesolomon** [bcgov/performance#207]

**Comment by phoebesolomon** [comment]
Employees are expected to sign-off on at least one official performance conversation with their supervisor every four months. If they want to meet more frequently, that is totally fine. However, they must meet at least once in every four month timeframe.

- [ ] Build in logic that checks most recent date of "past conversation" and displays a warning on the conversation tab if this "past conversation" was completed more than 4 months ago.

 

- [ ] Build in logic that checks most recent date of "past conversation" and displays a warning if employee tries to schedule a future conversation that is more than 4 months away.

Basically, if an employee hasn't had a conversation in 4 months, or if the employee tries to schedule a conversation for 10 months from now, I want the platform to catch that and say "You are expected to have a performance conversation once every four months, at minimum".

**Comment by phoebesolomon** [comment]
@Travis-A-Clark @Kousthubha-M see designs attached - let me know what you think. You will notice that if the user tries to schedule a conversation more than four months away, the calendar disables and then an alert pops up that explains why. Let me know what your thoughts are on this approach. 
![Performance Conversation Alert](https://user-images.githubusercontent.com/81586302/118051101-6ba92400-b34e-11eb-90eb-96c1aa295756.jpg)
![Disabled Conversation Month](https://user-images.githubusercontent.com/81586302/118051103-6c41ba80-b34e-11eb-8917-400026216068.jpg)

**Comment by Travis-A-Clark** [comment]
@phoebesolomon I really like the calendar concept when scheduling new conversations. It's really clear and aggressive in getting the message across. We'll see how users feel about it. :-)

As for the first screen, would it be possible to put the alert on the landing page for conversations rather than on the "Past Conversations" tab? I think it defaults to landing on "Upcoming Conversations", correct? So having it there instead.

In a future state, it is also the type of alert that I would anticipate showing up on the actual landing page after log-in where it welcomes the employee. For now, this is great. Thanks so much!

**Comment by phoebesolomon** [comment]
@Kousthubha-M @Travis-A-Clark see attached. 
![Performance Conversation Alert](https://user-images.githubusercontent.com/81586302/118191002-eb4bf700-b411-11eb-81cc-42481cb09ac5.jpg)
![Disabled Conversation Month](https://user-images.githubusercontent.com/81586302/118191004-eb4bf700-b411-11eb-9dd7-14961b2e59cb.jpg)
